src/Behat/Symfony2Extension/Context/Argument/ServiceArgumentResolver.php

```diff
@@ -108,15 +112,18 @@ private function getServiceName($argument)
     }
 
     /**
-     * @param ContainerInterface $container
-     * @param string $argument
-     * @return string
+     * Sanitise the container key given by the Behat config file,
+     * and retrieve the parameter from the Container.
+     *
+     * @param  ContainerInterface $container
+     * @param  string $key
+     * @return mixed
      */
-    private function replaceParameters(ContainerInterface $container, $argument)
+    private function replaceParameters(ContainerInterface $container, $key)
```

Comment:
No reason for changing the key, this PR was not in a ready state, still undergoing work as I was waiting for the test results from the CI process.

Has the behaviour of the method really changed? I don't think it has. I don't see the benefit of changing the name here, as it does still replace parameters, nothing new other than it will handle parameter collections.